x/auth/ante/ante_test.go

```diff
@@ -1034,25 +1034,6 @@ func (suite *AnteTestSuite) TestCustomSignatureVerificationGasConsumer() {
 			false,
 			sdkerrors.ErrInvalidPubKey,
 		},
-		{
-			"verify that an ed25519 account gets accepted",
-			func() {
-				priv1 := ed25519.GenPrivKey()
```

Comment:
I can create an issue to add back this test once we add ed25519 in the sdk.

Reply:
Yes, please do, and link a PR with a link to this code.

Reply:
https://github.com/cosmos/cosmos-sdk/issues/7352